src/main/java/edu/kis/vh/nursery/stack/IntArrayStack.java

```diff
@@ -20,7 +20,7 @@ public boolean isEmpty() {
 
     @Override
     public boolean isFull() {
-        return size == CAPACITY - 1;
+        return size == CAPACITY;
```

Comment:
Changing IS_EMPTY value requires such fixes? It does not make sense. Incorrect fix.

Reply:
When IS_EMPTY was -1, that meant that stack is full when size was 11. When IS_EMPTY was changed to 0 and all other implementation were fixed to do postincrementation, not preincrementation, stack if full when size is 12 (CAPACITY).